recipes/SpatialPy/meta.yaml

```diff
@@ -0,0 +1,63 @@
+
+{% set name = "spatialpy" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d4bf963ee707858dc8ff13b8f48dae18c4d6a0861965c525cc664ce098f676fd
```

Comment:
The version number was updated, but the source hash wasn't so the package isn't building. Please update the hash.